### ISSUE TITLE
Не удается организовать чтение сообщений

### ISSUE THREAD

**Issue by bf0rce** [BITERP/PinkRabbitMQ#18]
Версия компоненты 1.8.

Пример выполняемого кода.

```bsl
Потребитель = Клиент.BasicConsume(Параметры.Очередь, "", Истина, Ложь, 0);
// После этого вызова Потребитель = ""
// Второй параметр не влияет на дальнейший результат.

Пока Клиент.BasicConsumeMessage(Потребитель, ОтветноеСообщение, ТегСообщения, 1) Цикл
	// В цикл не заходит.
	ОтветноеСообщение = ""; // Обнуляем, чтобы избежать утечку памяти
	ТегСообщения = 0; // Обнуляем, чтобы избежать утечку памяти
КонецЦикла;
```

**Comment by ripreal** [comment]
Не является ошибкой. Потребитель - это не реализованный параметр API. (см документацию.)  Он всегда будет пустой строкой. А сообщения не приходят из очереди потому что слишком маленький таймаут в методе Клиент.BasicConsumeMessage. Рекомендуется ставить хотя бы 3000 (3 секунды) для маленьких сообщений.

**Comment by bf0rce** [comment]
Исправил так, но все равно не получает.
```bsl
Пока Клиент.BasicConsumeMessage("", ОтветноеСообщение, ТегСообщения, 30000) Цикл
```
В результате исполнение этой строки занимает 30 секунд и отладчик внутри цикла не останавливается. Подключенного потребителя на стороне RabbitMQ не видно.

Большее значение таймаута нецелесообразно, так как получение того же сообщения через HTTP занимает 69 мс.

**Comment by ripreal** [comment]
Значит в этой очереди нет сообщений, либо коннекта с сервером раббита нет. Нужно в веб-интерефейсе RAbbitMQ посмотреть, чтобы на закладках Connections и channels были активные элементы в момент вызова метода BasicConsumeMessage()

**Comment by bf0rce** [comment]
Да, есть там сообщение. Я одно забрал через HTTP (для проверки настроек соединения на форме), а второе компонента не забирает (с теми же настройками, что примечательно). И потребителя подключенного не видно.
![2020-02-11_12-25-11](https://user-images.githubusercontent.com/11785100/74224328-b0688a00-4cc9-11ea-9bd4-75cd6ebc617b.png)

Видимо, где-то есть какая-то опция, которая не позволяет компоненте зарегистрировать потребителя, что и ведет к 30 секундному таймауту при вызове метода BasicConsumeMessage().

Если интересно, я могу и обработку целиком прикрепить. Может быть, проблема где-то в другом месте моего кода.

**Comment by ripreal** [comment]
Да, нужна обработка. Желательно, чтобы в ней также программно создавались точки обмена и очереди с нужными свойствами

**Comment by bf0rce** [comment]
Обработка и скриншот.
[RabbitMQ_Console.zip](https://github.com/BITERP/PinkRabbitMQ/files/4210398/RabbitMQ_Console.zip)

![2020-02-16_17-16-32](https://user-images.githubusercontent.com/11785100/74609439-fcb52f00-50fa-11ea-8d47-d2ff4a7ad663.png)
